projects/distributed-tracing/src/shared/dashboard/dashboard-wrapper/navigable-dashboard.component.ts

```diff
@@ -37,10 +37,13 @@ import { GraphQlFilterDataSourceModel } from '../data/graphql/filter/graphql-fil
 })
 export class NavigableDashboardComponent implements OnChanges {
   @Input()
-  public readonly defaultJson?: ModelJson;
+  public navLocation?: string | null;
 
   @Input()
-  public navLocation?: string | null;
+  public defaultJson?: ModelJson;
+
+  @Input()
+  public variables?: Dictionary<unknown>;
```

Comment:
I generally prefer Map over a plain object, but I don't have strong preferences - would rather be consistent. Do you know if we use either as inputs elsewhere?

Reply:
I used Dictionary to pass in the request properties to table CDK (though we will eventually remove that and filters).

I kind of prefer Dictionary to Map because it doesn't need to be instantiated and is a little lighter weight. Can change though if we feel strongly about using Map instead.